### DISCUSSION TITLE
reindexing sensitivity?

### DISCUSSION THREAD

**Discussion by rhatguy** [bpatrik/pigallery2#338] [Q&A]
Is there a write up somewhere around what the different levels of reindexing sensitivity mean?  I.E. what changes if I select low, medium, or high?

**Comment by bpatrik** [comment]
This is the relevant code part:
https://github.com/bpatrik/pigallery2/blob/bae5559356e4d4320425415696e6fd6d24e5fc04/src/backend/model/database/sql/GalleryManager.ts#L36-L92

Its something like:
* low: only re-indexes if it detects a folder change
* medium: re-indexes every now and then
* high: re-indexes  at every request asynchronously (= returns with whatever is in the DB but refreshes the DB after that)

low should be fine for most of the use-cases
